docs/introduction/GettingStarted.md

```diff
@@ -39,16 +39,23 @@ yarn add @reduxjs/toolkit
 
 ### Create a React Redux App
 
-The recommended way to start new apps with React and Redux is by using the [official Redux+JS template](https://github.com/reduxjs/cra-template-redux) or [Redux+TS template](https://github.com/reduxjs/cra-template-redux-typescript) for [Create React App](https://github.com/facebook/create-react-app), which takes advantage of **[Redux Toolkit](https://redux-toolkit.js.org/)** and React Redux's integration with React components.
+The recommended way to start new apps with React and Redux is by using [our official Redux+TS template for Vite](https://github.com/reduxjs/redux-templates), or by creating a new Next.js project using [Next's `with-redux` template](https://github.com/vercel/next.js/tree/canary/examples/with-redux).
+
+Both of these already have Redux Toolkit and React-Redux configured appropriately for that build tool, and come with a small example app that demonstrates how to use several of Redux Toolkit's features.
 
 ```bash
-# Redux + Plain JS template
-npx create-react-app my-app --template redux
+# Vite with our Redux+TS template
+degit reduxjs/redux-templates/vite-template-redux
```

Comment:
What is `degit`? It's not introduced anywhere else in the doc, as far as I can tell.

This is probably better as a one-liner with `npx` (like below).

```suggestion
npx degit reduxjs/redux-templates/vite-template-redux
```

Reply:
Hah, yeah, haven't had a chance to actually _try_ these yet :)  I'll add `npx` and a comment clarifying what it does